Sources/Authentication/AuthenticationHandler.swift

```diff
@@ -21,11 +22,16 @@ import UIKit
 ///         - Login flow
 ///             - Get authorization code
 ///             - Fetch token using the authorization code
-public final class AuthenticationHandler: NSObject {
+public final class AuthenticationHandler: NSObject, ObservableObject {
     // MARK: Properties
-    internal let configuration: Configuration
-    internal let contextProvider: ASPresentationAnchor
     internal var sheetIsActive: Bool = false
+    internal var configuration: Configuration?
+    internal let contextProvider: ASPresentationAnchor?
```

Comment:
Why not put the contextProvider in the Configuration object so we only have one optional instead of two?

Reply:
I rather to keep it separated since those are not in the same context